### PR TITLE
Run on google collab

### PR DESCRIPTION
**Pull request by RomainGoussault**
Add notebook to run experiments on google collab with GPU

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -175,10 +181,18 @@ def run_scenario(current_scenario):
     return 0
 
 
-def get_config_from_file():
+def parse_command_line_arguments():
+
     parser = argparse.ArgumentParser()
     parser.add_argument("-f", "--file", help="input config file")
+    parser.add_argument("-v", "--verbose", help="verbose output", action="store_true")
```
Comment (bowni) [inline main.py:188]: Cool. Would be a good thing to add that in the README

**Comment by bowni** [comment]
Also, the first command in the notebook shouldn't includ the `-b collab` after the merge I suppose?

**Comment by natct10** [comment]
Did you see this about the versions: 
```sh
ERROR: tensorflow-probability 0.10.0 has requirement gast>=0.3.2, but you'll have gast 0.2.2 which is incompatible.
ERROR: google-colab 1.0.0 has requirement pandas~=1.0.0; python_version >= "3.0", but you'll have pandas 0.25.3 which is incompatible.
ERROR: datascience 0.10.6 has requirement folium==0.2.1, but you'll have folium 0.8.3 which is incompatible.
ERROR: albumentations 0.1.12 has requirement imgaug<0.2.7,>=0.2.5, but you'll have imgaug 0.2.9 which is incompatible.
```

Also, you have to be logged in to run the cells...

File: run_experiment_on_google_collab.ipynb [hunk 2]
```diff
@@ -0,0 +1,893 @@
+{
```
Comment (natct10) [inline run_experiment_on_google_collab.ipynb:1]: I am getting some errors in the "Analyse results" cell: 
```sh
# Read CSV results
df = pd.read_csv(experiment_path / "results.csv")
print(experiment_path)
```
No such file or directory: 'experiments'

Comment (RomainGoussault) [reply run_experiment_on_google_collab.ipynb:1]: experiment folder is created when the `main.py` file is run. Did it run ?

**Comment by RomainGoussault** [comment]
> Also, the first command in the notebook shouldn't includ the `-b collab` after the merge I suppose?

yes

File: README.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ Finally, with given scenarios and multi-partner learning approaches, we can addr
 
    Under `scenario_params_list`, enter a list of sets of scenario(s). Each set starts with ` - dataset_name:` and must have only one `partners_count` value. The length of `amount_per_partners`, `corrupted_datasets` (and `samples_split_option` when the advanced definition is used) must match the `partner_counts` value. If for a given parameter multiple values are specified, e.g. like for `agregation_weighting` in the first scenario set of the above example, all possible combinations of parameters will be assembled as separate scenarios and run.
    
-3. Then execute `main.py -f config.yml`
+3. Then execute `main.py -f config.yml`. Add the `-v` argument if you want a more verbose output.
```
Comment (natct10) [inline README.md:118]: :+1:

File: run_experiment_on_google_collab.ipynb (hunk 2 above)
Comment (natct10) [reply run_experiment_on_google_collab.ipynb:1]: Yes!

**Pull request merged**